app/auth/auth_service.ts

```diff
@@ -103,6 +104,10 @@ export class AuthService {
   }
 
   handleLoggedIn(response: user.GetUserResponse) {
+    if (window.opener) {
+      console.log("Posting success message to " + window.location.origin);
+      window.opener?.postMessage("", window.location.origin);
```

Comment:
nit

```suggestion
      window.opener.postMessage("", window.location.origin);
```

Reply:
I removed the log statements per Jim's suggestions, so the if became useless - so I kept the question mark